src/data/panel.ts

```diff
@@ -44,3 +44,39 @@ export const getPanelTitle = (hass: HomeAssistant): string | undefined => {
 
   return hass.localize(`panel.${panel.title}`) || panel.title || undefined;
 };
+
+export const getPanelText = (hass: HomeAssistant, panel: Panels["panel"]) => {
```

Comment:
Combine this with `getPanelTitle` no need to have the same logic twice

Reply:
So you're saying to change `getPanelTitle` from something that returns a title to something that returns an object? 

Something like this:

```
interface PanelStrings {
  title: string;
  text: string;
  icon: string;
}

export const getPanelStrings = (hass: HomeAssistant): PanelStrings => {
```

Reply:
No, bring this code to a seperate function that both functions use:
```
  if (!panel) {
    return undefined;
  }
  if (panel.url_path === "lovelace") {
    return hass.localize("panel.states");
  }
  if (panel.url_path === "profile") {
    return hass.localize("panel.profile");
  }

  return hass.localize(`panel.${panel.title}`) || panel.title || undefined;
```

Reply:
Ah okay. Got it

Reply:
Done... I guess :P It wasn't a straight-forward copy/paste to separate function since `getPanelTitle` is using `panel.url_path` to determine translationKey, but `getPanelText` was using a combination of `panel.title` and `panel.component_name`.